node-backend/app/controllers/orgnizationController.js

```diff
@@ -0,0 +1,165 @@
+const db = require("../models/sequelize");
+const Organizations = db.organizations;
+const yup = require("yup");
```

Comment:
yup should be included at top then give new line and then add your code modules. Third party modules and our code modules must be separated and  grouped